dot/sync/disjoint_block_set_integeration_test.go

```diff
@@ -60,12 +56,13 @@ func TestDisjointBlockSet(t *testing.T) {
 	s.addHeader(header2)
 	require.Equal(t, 3, s.size())
 	expected = &pendingBlock{
-		hash:    header2.Hash(),
-		number:  header2.Number,
-		header:  header2,
-		clearAt: time.Time{}.Add(ttl),
+		hash:   header2.Hash(),
+		number: header2.Number,
+		header: header2,
 	}
-	require.Equal(t, expected, s.getBlock(header2.Hash()))
+	block2 := s.getBlock(header2.Hash())
+	block2.clearAt = time.Time{}
```

Comment:
we should assert the `clearAt` field somehow (even if it's not an exact assertion, at least verify it's set in the future or something)